src/infrastructure/jira/inbound-auth/jira-asymmetric-jwt-token-verifier.ts

```diff
@@ -72,45 +71,37 @@ export class JiraAsymmetricJwtTokenVerifier {
 			query?: Record<string, unknown>;
 		},
 	): Promise<void> => {
-		try {
```

Comment:
**note:** While diff looks a quite significant, there is no logical changes in this file. The PR:

- Removes `try/catch` to let the error propagate further. 
- Throws `Error` instead of a custom error. **All** errors from this module are mapped to `UnauthorizedResponseStatusError` in the corresponding middleware to avoid exposing auth-related details. If we need at some point a more granular error handling, we can introduce additional errors specific to this module.